src/main/groovy/org/pih/warehouse/inventory/CycleCountStatus.groovy

```diff
@@ -0,0 +1,14 @@
+package org.pih.warehouse.inventory
+
+enum CycleCountStatus {
```

Comment:
I'll let Artur review these since he did the bulk of the work on statuses but do we need both CREATED and REQUESTED?

It's probably worthwhile adding docstrings to each of the statuses to describe what they mean. Some are obvious, but still probably helpful for future us to understand.

Reply:
I consulted it with Artur before, and we agreed that for now we can go the easiest way possible and elaborate on it either in the code review or refactor it later, since this wouldn't be a difficult change.